### PR TITLE
Fix pod toleration test case, add new TC for unsupported tolerations

### PR DESCRIPTION
**Pull request by lugi0**
Signed-off-by: Luca Giorgi <lgiorgi@redhat.com>

### CODE REVIEW

File: tests/Tests/400__ods_dashboard/415__ods_dashboard_settings_pod_tolerations.robot [hunk 1]
```diff
@@ -6,13 +6,25 @@ Suite Setup      Begin Web Test
...
 *** Test Cases ***
-Test Setting Pod Toleration Via UI
+Test Setting Unsupported Pod Toleration Via UI
     [Documentation]    Sets a Pod toleration via the admin UI
     [Tags]  Sanity    Tier1
-    ...     ODS-1684
+    ...     ODS-1788
     Menu.Navigate To Page    Settings    Cluster settings
     Wait Until Page Contains    Notebook pod tolerations
+    Set Pod Toleration Via UI    --UNSUPPORTED--
```
Comment (jgarciao) [inline tests/Tests/400__ods_dashboard/415__ods_dashboard_settings_pod_tolerations.robot:20]: If you want to enhance it a little bit more, consider creating a keyword `Unsupported Pod Toleration Should Not Be Allowed` (or similar) wrapping lines 20-22 and then calling it with different unsupported values

**Review by jgarciao — APPROVED** [review]
lgtm (consider optional suggested improvement)

File: tests/Tests/400__ods_dashboard/415__ods_dashboard_settings_pod_tolerations.robot (hunk 1 above)
Comment (lugi0) [reply tests/Tests/400__ods_dashboard/415__ods_dashboard_settings_pod_tolerations.robot]: Done

File: tests/Tests/400__ods_dashboard/415__ods_dashboard_settings_pod_tolerations.robot [hunk 2]
```diff
@@ -6,21 +6,36 @@ Suite Setup      Begin Web Test
...
 
 
+*** Variables ***
+${TOLERATION_CHECKBOX}=    //input[@id="tolerations-enabled-checkbox"]
+<<<<<<< HEAD
+@{UNSUPPORTED_TOLERATIONS}=    --UNSUPPORTED--    Unsupported-    -Unsupported    Unsupported!    1-_.a@    L@5t0n3!
+=======
+>>>>>>> b3aed3a (Rename file, test unsupported toleration, fix test by increasing sleep)
+
+
```
Comment (bdattoma) [inline tests/Tests/400__ods_dashboard/415__ods_dashboard_settings_pod_tolerations.robot:16]: maybe from a conflict resolution

Comment (lugi0) [reply tests/Tests/400__ods_dashboard/415__ods_dashboard_settings_pod_tolerations.robot:16]: good catch

**Review by tarukumar — APPROVED** [review]
lgtm

**Review by bdattoma — APPROVED** [review]

**Pull request merged**